### PR TITLE
Add `batch_norm` opmapper

### PR DESCRIPTION
**Pull request by thisjiang**
As title.

### CODE REVIEW

File: cinn/frontend/op_mappers/batchnorm.cc [hunk 1]
```diff
@@ -47,11 +47,98 @@ void BatchnormOpMapper(const paddle::cpp::OpDesc& op_desc, const OpMapperContext
   ctx.AddVarModelToProgram(out_name, out->id);
 }
 
+void BatchNormTrainOpMapper(const paddle::cpp::OpDesc& op_desc, const OpMapperContext& ctx) {
```
Comment (CtfGo) [inline cinn/frontend/op_mappers/batchnorm.cc]: 可以用OpDesc中的is_test属性判断是否推理算子，由此决定调用builder的batchnorm还是batch_norm_train接口，不需要新加OpMapper

File: cinn/frontend/op_mappers/batchnorm.cc [hunk 2]
```diff
@@ -53,5 +124,6 @@ void BatchnormOpMapper(const paddle::cpp::OpDesc& op_desc, const OpMapperContext
 
 CINN_REGISTER_HELPER(batchnorm) {
   CINN_REGISTER_OP_MAPPER(batchnorm, cinn::frontend::op_mappers::BatchnormOpMapper)
```
Comment (Xreki) [inline cinn/frontend/op_mappers/batchnorm.cc]: 注册这里用的key，需要跟Paddle的op_type保持一致吗？

Comment (thisjiang) [reply cinn/frontend/op_mappers/batchnorm.cc]: 需要，不然Paddle符号执行的时候找不到

File: cinn/frontend/op_mappers/batchnorm.cc (hunk 1 above)
Comment (thisjiang) [reply cinn/frontend/op_mappers/batchnorm.cc]: 已修改

**Review by wzzju — DISMISSED** [review]
LGTM

File: cinn/frontend/op_mappers/batchnorm.cc (hunk 2 above)
Comment (Xreki) [reply cinn/frontend/op_mappers/batchnorm.cc]: Paddle中注册的op_type是`batch_norm`、`batch_norm_grad`：
https://github.com/PaddlePaddle/Paddle/blob/0a963ee9211174766dd4f718b43f9965b467cd4b/paddle/fluid/operators/batch_norm_op.cc#L1279

Comment (thisjiang) [reply cinn/frontend/op_mappers/batchnorm.cc]: 唉？我看错了么。。。我改下

**Review by CtfGo — COMMENTED** [review]
LGTM

**Review by CtfGo — DISMISSED** [review]

**Review by wzzju — APPROVED** [review]
LGTM
